### PR TITLE
Implementation Pattern AAA

### PR DESCRIPTION
**Pull request by lufernando100**
Implementation of Pattern AAA

### CODE REVIEW

File: cypress/e2e/dresses-list.cy.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  let menuContentPage: MenuContentPage;
+  let dressesListPage: DressesListPage;
+  let dressNumber: number;
+  let dressName: Array<string>;
+
+  before(() => {
+    menuContentPage = new MenuContentPage();
+    dressesListPage = new DressesListPage();
+    menuContentPage.visitMenuContentPage();
+    dressNumber = 5;
```
Comment (juanjo44) [inline cypress/e2e/dresses-list.cy.ts]: it's very specific, you can do it more general. for example imagine a situation where you want to add a new dress name. you can use the length of the dressName array

Comment (lufernando100) [reply cypress/e2e/dresses-list.cy.ts]: It was fixed

**Comment by manuelq12** [comment]
Why didn't you change nothing on the buy-tshirt.cy.ts? Does AAA is already there?

**Comment by lufernando100** [comment]
> Why didn't you change nothing on the buy-tshirt.cy.ts? Does AAA is already there?

I have added new changes for the buy test

**Review by manuelq12 — APPROVED** [review]
LGTM

**Pull request merged**